tests/components/landisgyr_heat_meter/test_sensor.py

```diff
@@ -72,7 +74,7 @@ async def test_create_sensors(
     await async_setup_component(hass, HA_DOMAIN, {})
     await hass.async_block_till_done()
 
-    # check if 26 attributes have been created
+    # check if the right number of attributes have been created
```

Comment:
I think maybe you should look at `SnapshotAssertion`

https://github.com/home-assistant/core/blob/c507ca1e665abffd9b8986f0974f74d35cde1653/tests/components/sfr_box/test_sensor.py#L38-L41

and

https://github.com/home-assistant/core/blob/c507ca1e665abffd9b8986f0974f74d35cde1653/tests/components/sfr_box/test_sensor.py#L49-L50

Reply:
I will. This also sounds like something for a seperate PR. What do you think?

Reply:
Yes - most likely a separate PR. I'm not sure if it should be preliminary or follow-up...

Reply:
I'll start working on the PR asap. If this PR has to wait for that I understand. If not, it will surely please some users :)

Reply:
I've created #90126. I'm curious if this is what you had in mind. If so, We can choose which PR goes first.

Reply:
I merged #90126. This will need to be rebased accordingly.

Reply:
Thanks. I've done the rebase. I'll pick up the other suggestions later today.